dom/manipulate.js

```diff
@@ -33,23 +33,40 @@ function parseHtml (html)
  * @returns {HTMLElement}
  */
 export function createElement (type, attributes = {})
+{
+    const element = createUnstyledElement(type, attributes);
+
+    if (attributes.css !== undefined)
+    {
+        setStyles(element, attributes.css);
```

Comment:
Code-Splitting with `required()` would be totally ace here, but currently very hard to implement in our projects.